benchmark/kernel/base.rb

```diff
@@ -5,21 +5,46 @@
 BENCHMARKS = File.join(__dir__, "src")
 
 class KernelBenchmark
+    def load_map
+        Zemu::Debug.load_map("kernel_debug.map") do |s|
```

Comment:
This parsing block is duplicated between here and the integration tests. It should be refactored.